theseus/core/objective.py

```diff
@@ -471,9 +476,21 @@ def _get_batch_size(batch_sizes: Sequence[int]) -> int:
         batch_sizes.extend([v.data.shape[0] for v in self.aux_vars.values()])
         self._batch_size = _get_batch_size(batch_sizes)
 
+    def update_vectorization(self):
+        if self._vectorization_run is None:
+            return
+        if self._batch_size is None:
+            self.update()
+        self._vectorization_run()
+
```

Comment:
Nit to add explicit `return None`
```
if self._vectorization_run is not None:
        if self._batch_size is None:
            self.update()
        self._vectorization_run()
return None
```

Reply:
Your code looks cleaner, but I'm a bit confused by the explicit `return None`. Why do we need that line?

Reply:
The explicit return None seems like a [good practice thing](https://realpython.com/python-return-statement/#using-the-python-return-statement-best-practices) and somewhat a convention. I can't remeber if there are other place in our code where we have functions not explicitly returning anything (aside from the mock ones that use pass).